src/utils/get-notification-channels.ts

```diff
@@ -0,0 +1,21 @@
+import config from "../config";
+
+/**
+ * Returns the appropriate Discord webhook URL secret ID and Telegram chat ID
+ * based on whether the application is in development or production mode.
+ */
+export default function getNotificationChannels() {
+  const isDevelopment = process.env.NODE_ENV === "development";
+
+  return {
+    discordWebhookUrlSecretId:
+      isDevelopment && config.DISCORD_TEST_WEBHOOK_URL_SECRET_ID
```

Comment:
A downside of this check is that if one forgets to set the id and runs `npm test` it will send multiple notifications to the prod discord/telegram channels (just happened to me). Maybe we can have default test channels or make configuring these mandatory?

Reply:
good point, will make those env vars mandatory